test/webapp_tests.js

```diff
@@ -306,6 +306,14 @@ test('webapp tests', async(t) => {
     await createApiKey(request, account_sid);
     await createApiKey(request, account_sid);
 
+    // /* retrieve list of users */
```

Comment:
why is the test commented out?  We would want a test for this new api, no?

Reply:
@davehorton wanted to show you. Not sure why the the test fails to be unauthorized. So something here doesnt work with jwt. But I was not able to figure out why, since all other tests use it the same way.

Reply:
ah, ok will have a look..

Reply:
yeah the problem is the jwt you are using has only account level scope.  It was created further up in the test suite [here](https://github.com/jambonz/jambonz-api-server/blob/8d303390b7c3e003ad847dd44f4467402116196c/test/webapp_tests.js#L34), using a POST /register api call, which is currently only used on the hosted platform (jambonz.xyz and jambonz.us) where all users have account-level permissions.

Reply:
@davehorton can I make changes to that or better move the test to a different file that tests with different permissions? We could also make a new file to test user permissions since there will be more things to test regarding that later.